format.js

```diff
@@ -45,6 +45,53 @@ function format(FORMAT, now) {
             var vulgar_fraction = ["\u2070/\u2080", "\u00B9/\u2084", "\u00B9/\u2082", "\u00B3/\u2084", "\u00B9/\u2081"][quarters];
             desired = desired.replace(/%;vf/g, vulgar_fraction);
         }
+        if (FORMAT.indexOf("%;nf") > -1) {
+            var quarters = Math.round(now.get_minute() / 15);
+            
+            var hour = now.get_hour();
+            
+            // convert from 0-23 to 1-12
+            if (hour > 12) {
+                hour -= 12;
+            }
+            if (hour == 0) {
+                hour = 12;
+            }
+            
+            // convert from 0-23 to 1-12
```

Comment:
Code duplication, would be better to create a function that does the 0-23 to 1-12 conversion.